roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -99,13 +99,32 @@
     - "{{ ceph_keys.get('stdout_lines') | default([]) }}"
   when: cephx
 
+- name: check if libvirt keyring exists
+  command: find /etc/ceph -name "*.libvirt.keyring"
```

Comment:
@leseb Yes, you are right. Wildcard ideally wouldn't work with command module. I have to use the shell module for the command, take the output in a register variable, then use command module. Then it should work.